packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -314,10 +314,7 @@ export default class UserStorageController extends BaseController<
       );
     },
     doesInternalAccountHaveCorrectKeyringType: (account: InternalAccount) => {
-      return (
-        account.metadata.keyring.type === KeyringTypes.hd ||
-        account.metadata.keyring.type === KeyringTypes.simple
-      );
+      return account.metadata.keyring.type === KeyringTypes.hd;
```

Comment:
So thaaats what `.simple` is lol.
OK so we are now POSITIVE this is only wallet accounts?

Reply:
Yes, learnt the hard way 😂 
WE ARE! 🚀